core/src/main/java/com/google/zxing/oned/rss/expanded/RSSExpandedReader.java

```diff
@@ -249,9 +248,10 @@ private List<ExpandedPair> checkRows(List<ExpandedRow> collectedRows, int curren
   }
 
   // Whether the pairs form a valid finder pattern sequence, either complete or a prefix
-  private static boolean isValidSequence(List<ExpandedPair> pairs) {
+  private static boolean isValidSequence(List<ExpandedPair> pairs, boolean complete) {
 
     for (int[] sequence : FINDER_PATTERN_SEQUENCES) {
+      boolean sizeOk = (complete ? pairs.size() == sequence.length : pairs.size() <= sequence.length);
```

Comment:
This var isn't used, that's why codacy is complaining

Reply:
Oops, missed a line. Fix incoming!